workers/shared/processing/files/processor.py

```
@@ -597,37 +607,49 @@ def process_file(
-                        logger.error(
-                            f"Failed to update file execution status: {status_error}"
+                        try:
+                            context.api_client.update_file_execution_status(
+                                file_execution_id=context.workflow_file_execution_id,
+                                status=ExecutionStatus.ERROR.value,
+                                error_message=error_msg,
+                            )
+                            logger.info(
+                                f"Updated file execution {context.workflow_file_execution_id} status to ERROR"
+                            )
+                        except Exception as status_error:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)